engine/src/test/java/com/redhat/cloud/notifications/templates/TestInventoryTemplate.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.InventoryTestHelpers;
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.templates.models.Environment;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
```

Comment:
It's missing this import

```suggestion

import javax.inject.Inject;

```